### ISSUE TITLE
The link at "see benchmarks below" doesn't work

### ISSUE THREAD

**Issue by jkleiser** [localvoid/uix#13]
The link at "see benchmarks below" (#benchmarks) doesn't work here: https://pub.dartlang.org/packages/uix
However, it works here: https://github.com/localvoid/uix

**Comment by localvoid** [comment]
Yes, markdown to html compiler seems to be broken on pub.dartlang.org ,it also removed comments in code sections.
